src/Transport/Config/AzureServiceBusTransportInfrastructure.cs

```diff
@@ -14,6 +15,11 @@ class AzureServiceBusTransportInfrastructure : TransportInfrastructure
         ITopology topology;
         SatelliteTransportAddressCollection satelliteTransportAddresses;
 
+        public override Task Stop()
+        {
+            return ((IStoppableTopology) topology).Stop();
```

Comment:
right, we'd have to expose the container from the topology, that may not be a good idea